### PR TITLE
Bug 1174231 - Compare chooser shows red when validation has not failed

### PR DESCRIPTION
**Pull request by MikeLing**
[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/626)

### CODE REVIEW

File: ui/partials/perf/comparechooserctrl.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
           <label for="original-project-selector">Project</label>
           <select id="original-project-selector" class="form-control" ng-model="originalProject" ng-options="project.name for project in projects"/>
           <label for="original-revision-input">Revision</label>
-          <div class="form-group has-error">
+          <div class="form-group" ng-class="{'has-error':originalRevisionError}">
```
Comment (tojonmz) [inline ui/partials/perf/comparechooserctrl.html]: This should have a single character white space after the `'has-error:`

File: ui/partials/perf/comparechooserctrl.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
           <label for="new-project-selector">Project</label>
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
-            <div class="form-group has-error">
+            <div class="form-group" ng-class="{'has-error':newRevisionError}">
```
Comment (tojonmz) [inline ui/partials/perf/comparechooserctrl.html]: Same here.

**Comment by tojonmz** [comment]
Thanks @MikeLing ! This looks fine in local testing on:

OSX 10.10.3
FF Nightly **41.0a1 (2015-06-12)**
Chrome Release **43.0.2357.124 (64-bit)**

A reminder to add the PR as an r? text-only bug attachment when you need a review. Thanks!

**Comment by wlach** [comment]
Looks good except the issues @tojonmz just mentioned. And yes, please do create an attachment and ask for review.

File: ui/partials/perf/comparechooserctrl.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
           <label for="original-project-selector">Project</label>
           <select id="original-project-selector" class="form-control" ng-model="originalProject" ng-options="project.name for project in projects"/>
           <label for="original-revision-input">Revision</label>
-          <div class="form-group has-error">
+          <div class="form-group" ng-class="{'has-error' : originalRevisionError}">
```
Comment (wlach) [inline ui/partials/perf/comparechooserctrl.html]: You added a space before and after the : where you should have only added one after the colon. Please change:

{'has-error' : originalRevisionError} -> {'has-error': originalRevisionError}

File: ui/partials/perf/comparechooserctrl.html [hunk 4]
```diff
@@ -23,7 +23,7 @@
           <label for="new-project-selector">Project</label>
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
-            <div class="form-group has-error">
+            <div class="form-group" ng-class="{'has-error' : newRevisionError}">
```
Comment (wlach) [inline ui/partials/perf/comparechooserctrl.html]: Same here

**Pull request merged**